portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,29 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
+    <div id="titulo">
+      <h1>Nicolas's Portfolio</h1>
       <p>This is my portfolio.</p>
+    </div>
+    <div id="contenido">
+      <p>Hi, i'm Nicolas, i'm from Colombia and I am 18 years old. Right now i'm living in Monterrey, Mexico since i'm in my 2nd year of university. 
+    I like a lot to learn new things (mostly computer based things) and also I love to do a lot of exercise.
+    I've been coding since my fist year of Highschool (15 y/o).</p>
+
+
+    <div class="row">
+        <div class="column">
+            <p align="center" style="font-size: 90%;">This is a picture of me in <br> Guatemala on vacations.</p>
+        <img src="/images/yo.jpeg" class="yo"/>
+        </div>
+        <div class="column">
+            <p align="center" style="font-size: 90%;">This is a picture of my cute cat, <br>his name is Paul.</p>
+        <img src="/images/paul.jpeg" class="paul"/>
+        </div>
+    </div>
       <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <p>Click here to get a random fact about my self :)</p>
```

Comment:
Something like:

```suggestion
      <p>Click the button below to get a random fact about my self :)</p>
```

may be slightly clearer